project/LinkerdBuild.scala

```diff
@@ -65,15 +67,57 @@ object LinkerdBuild extends Base {
       .aggregate(core, http, mux, thrift)
   }
 
+  val configCore = projectDir("config")
```

Comment:
TIOLI: I have a slight preference for calling this project val `config` instead of `configCore`. `configCore` implies to me that there's a `core` sub-project inside the `config` project, which is not the case.